### PR TITLE
Icon correction wrong

### PR DESCRIPTION
**Pull request by Fausto95**
Capture d’écran 2019-03-22 à 16 50 45

- [ ] Already covered by tests
- [ ] Manual testing
- [x] Unit testing

### CODE REVIEW

**Comment by Fausto95** [comment]
[Bug UX popin interquestion failed](https://trello.com/c/oGwwmtga/1189-bug-ux-popin-interquestion-failed)

File: flow-typed/coorpacademy_nova_icons.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ declare module "@coorpacademy/nova-icons" {
     declare var NovaLineVideosVideoClip3: typeof NovaIcons;
     declare var NovaLineFilesOfficeFileOfficePdf: typeof NovaIcons;
     declare var NovaLineComputersComputerScreen1: typeof NovaIcons;
+    declare var NovaLineStatusCloseCircle: typeof NovaIcons;
```
Comment (VincentCATILLON) [inline flow-typed/coorpacademy_nova_icons.js]: 👍

File: src/components/card-header.js [hunk 2]
```diff
@@ -59,30 +58,43 @@ const styles = StyleSheet.create({
...
-    </Text>
-  </View>
-);
+const CORRECTION_BG_COLOR = {
+  CORRECT: {backgroundColor: '#d3f1e2'},
+  WRONG: {backgroundColor: '#FDD9DC'}
+};
+
+const CardHeader = ({type, isCorrect, title}: Props) => {
+  const correctionBackgroundColor =
```
Comment (VincentCATILLON) [inline src/components/card-header.js]: Use style for this please (for cache perf)

Comment (Fausto95) [reply src/components/card-header.js]: https://github.com/facebook/react-native/blob/8eed600d5b2be78c405f7ef139a7b0e4a2480c85/Libraries/StyleSheet/StyleSheet.js#L69

Comment (Fausto95) [reply src/components/card-header.js]: There's no cache related to this API

Comment (VincentCATILLON) [reply src/components/card-header.js]: Inspect your code, `StyleSheet` API don't transport every styles as object, but as index, as a classes in CSS: 

https://stackoverflow.com/questions/39336266/react-native-inline-styles-and-performance

**Pull request merged**